docs/ru/README.md

```diff
@@ -52,17 +53,26 @@
 
 После установки веб-интерфейс (loghouse-dashboard) будет доступен по домену, указанному в конфигурационном файле values.yaml как ```loghouse_host```. Потребуется пройти базовую аутентификацию, сгенерированную с помощью htpasswd и определенную в параметре ```auth``` конфигурационного файла values.yaml.
 
+# Обновление
+
+При переходе на версию **0.3** Helm chart претерпел некоторые изменения. Мы изменили работу с хуками, а так же обновили версии API у объектов Kubernetes. Чтобы обновиться с предыдущей версии надо удалить активные job, а так же ingress. Сделать это можно вот такой командой
+```
+kubectl -n loghouse delete jobs,ing --all
+```
+
+**Важно знать!** Новая версия изменит структуру базы и некоторое время будет переносить данные в новые таблицы. Если ваши логи вам ценны - сделайте бэкап перед обновлением.
```

Comment:
**Warning!** Database schema was changed. Please, create a backup of clickhouse data before upgrade. Note that the migration task will automatically start at the end of the upgrade process.

```suggestion
**Важно знать!** Схема базы данных изменилась, поэтому обязательно сделайте бэкап данных в clickhouse перед обновлением. В процессе обновления будет запущена миграция, которая изменит структуру базы и некоторое время будет переносить данные в новые таблицы. 
```